Core/Services/StitchCoreRemoteMongoDBService/Sources/StitchCoreRemoteMongoDBService/Sync/ChangeEvent.swift

```diff
@@ -0,0 +1,168 @@
+import MongoSwift
+
+/// The type of operation that occurred within a ChangeEvent.
+public enum OperationType: String, Codable {
+    case insert, delete, replace, update, unknown
```

Comment:
Invalidate?

Reply:
ah discussed with doug offline. `invalidate` is an event type that exists for MongoDB change events, but it doesn't exist for us since we close the client stream when we get an invalidate event on the Stitch server, so we don't need it. My other comment about the unknown event still stands though